blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServer.scala

```diff
@@ -125,51 +125,50 @@ class BlazeBuilder(
   def start: Task[Server] = Task.delay {
     val aggregateService = Router(serviceMounts.map { mount => mount.prefix -> mount.service }: _*)
 
-    val pipelineFactory = getContext() match {
-      case Some((ctx, clientAuth)) =>
-        (conn: SocketConnection) => {
-          val eng = ctx.createSSLEngine()
-          val requestAttrs = {
-            var requestAttrs = AttributeMap.empty
-            (conn.local,conn.remote) match {
-              case (l: InetSocketAddress, r: InetSocketAddress) =>
-                requestAttrs = requestAttrs.put(Request.Keys.ConnectionInfo, Request.Connection(l,r, true))
+    def resolveAddress(address: InetSocketAddress) =
```

Comment:
I was just taking things apart while I found the repetition.  It could be inlined again.